### PR TITLE
Maintenance

### PR DESCRIPTION
**Pull request by tessi**
* moved to CircleCI
* upgraded gems (security upgrade of nokogiri is involved, also rubocop)
* bumped version to 0.2.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+* 
+
+
+## 0.3.0 / 2019-08-21
+
+### Security
+
+* upgrade nokogiri (addressed CVE-2019-5477)
+* upgrade all dependent gems
+* moved to CircleCI
```
Comment (pmeinhardt) [inline CHANGELOG.md]: In all but the last bullet points, you use present tense… 🤓

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+          name: rubocop
+          command: |
+            bundle exec rubocop
+      - run:
+          name: Wait for DB
+          command: dockerize -wait tcp://localhost:5432 -timeout 1m
+      - run:
+          name: Database Setup
+          command: |
+            bundle exec rails db:drop || true
```
Comment (pmeinhardt) [inline .circleci/config.yml]: Same here: I don't think it's necessary to drop the database.

File: CHANGELOG.md (hunk 1 above)
Comment (tessi) [reply CHANGELOG.md]: 👍 fixed

File: .circleci/config.yml (hunk 2 above)
Comment (tessi) [reply .circleci/config.yml]: good spoeeted 👍  fixed here and in that other PR

**Pull request merged**